src/View/Helper/TextHelper.php

```diff
@@ -167,7 +168,7 @@ protected function _insertPlaceHolder($matches)
         if (isset($matches['url_bare'])) {
             $match = $matches['url_bare'];
         }
-        $key = md5($match);
+        $key = hash_hmac('sha1', $match, Security::getSalt());
```

Comment:
Would using a salted sha1 or sha256 work here?

Reply:
it would, need to change that?

Reply:
No. I don't forsee this becoming a performance issue.